src/Cli/dotnet/ReleasePropertyProjectLocator.cs

```diff
@@ -186,15 +185,16 @@ private bool IsValidProjectFilePath(string path)
         }
 
         /// <returns>A case-insensitive dictionary of any properties passed from the user and their values.</returns>
-        private Dictionary<string, string> GetGlobalPropertiesFromUserArgs(IEnumerable<string> userPropertyArgs)
+        private Dictionary<string, string> GetGlobalPropertiesFromUserArgs(ParseResult parseResult)
         {
             Dictionary<string, string> globalProperties = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
-            IEnumerable<(string key, string value)> globalPropEnumerable = MSBuildPropertyParser.ParseProperties(String.Join(";", userPropertyArgs));
 
-            // The parser puts keys into the format --property:Key no matter the user input, so we can expect that pattern and extract the key.
-            foreach (var kvPair in globalPropEnumerable)
+            string[] globalPropEnumerable = parseResult.GetValueForOption(CommonOptions.PropertiesOption);
+
+            foreach (var keyEqVal in globalPropEnumerable)
             {
-                globalProperties[kvPair.key.Split(new string[] { "--property:" }, StringSplitOptions.None).Last()] = kvPair.value;
+                string[] keyValuePair = keyEqVal.Split("=");
+                globalProperties[keyValuePair.First()] = keyValuePair.Last();
```

Comment:
I'd suggest indexing directly into the array here:

```suggestion
                globalProperties[keyValuePair[0]] = keyValuePair[1];
```